Issue 99

suryadipnag opened the pull request:
# Pull Request Review

## Checklist

- [x] **Issue** : https://github.com/IBM/openstack-vim-driver/issues/99
- [x] **List of related PRs** : N.A.
- [x] **Project builds without any errors** : N.A.
- [x] **Unit Tests** : N.A.
- [x] **Ran manual functional “smoke” test (does product as a whole still work?)** : N.A.
- [x] **User Story meets the acceptance criteria and passes**
- [x]  **Description of the changes**:  Updating install instructions for resource drivers as they are outdated. Tested the updated helm version 3 commands for installing the openstack-vim driver helm package and it is working fine.

suryadipnag commented:
This install command is for Helm v3 CLI. I have tested it in OCP. Before running this install command I have checked the Helm version and then executed the Helm Install command. Below is the snapshot for the same.

![image](https://user-images.githubusercontent.com/106679486/184812585-1bf7813d-ca14-429d-b55f-17bc8c044d3b.png)

suryadipnag commented:
@dvaccarosenna , Now I have added the helm version in the doc.

lokanalla reviewed (changes requested):
Hi @suryadipnag ,

swagger-ui is removed from ignition and I see a reference in the vim driver docs. You might need to remove it.

suryadipnag commented:
> Hi @suryadipnag ,
> 
> swagger-ui is removed from ignition and I see a reference in the vim driver docs. You might need to remove it.

Hi @lokanalla , I have now removed the swagger reference and added the health check api for checking the access for the Driver like we had it already for Ansible and Kubernetes Driver.
